test/channelSpec.js

```diff
@@ -231,6 +258,7 @@ describe('makePaymentChannelPlugin', function () {
           try {
             assert.deepEqual(ctx.state, {})
             assert.equal(ctx.plugin, this.plugin)
+            // TODO: Decide on the claim side-protocol
```

Comment:
claim as passed into handleIncomingClaim should already be inside the side protocol